go/libkb/pseudonym.go

```diff
@@ -19,18 +20,15 @@ import (
 // TLFPseudonym is an identifier for a key in a tlf
 type TlfPseudonym [32]byte
 
-type keyGen int
-type tlfID [16]byte
-
 const tlfPseudonymVersion = 1
 
 // TlfPseudonymInfo is what a pseudonym represents
 type TlfPseudonymInfo struct {
 	// TLF name like: /keybase/private/a,b
 	Name string
 	// TLF id
-	ID      tlfID
-	KeyGen  keyGen
+	ID      [16]byte
+	KeyGen  int
```

Comment:
These need to be public types, or else engine code can't instantiate this struct. I went with generic types, but we could also make these public instead of deleting them?